.github/pull_request_template.md

```diff
@@ -3,33 +3,32 @@
 
 ### What is changed and how does it work?
 
-### Check List
+### Checklist
 <!-- Remove the items that are not applicable. -->
 
 Tests
 <!-- At least one of them must be included. -->
 
-- Unit test
-- E2E test
-- Manual test (add detailed scripts or steps below)
-- No code
+- [ ] Unit test
+- [ ] E2E test
+- [ ] Manual test (add detailed scripts or steps below)
+- [ ] No code
 
 Code changes
 
-- Has Go code change
-- Has CI related scripts change
-- Has Terraform scripts change
+- [ ] Has Go code change
+- [ ] Has CI related scripts change
+- [ ] Has Terraform scripts change
```

Comment:
I think we can remove all these `Code changes`. They are not really necessary because what are changed is obvious. And we don't really have terraform scripts. What do you think :)?

Reply:
I think so. All these three changes are obvious in the code. What about @zhouqiang-cl @cwen0 ?